payslip_line_manually/models/hr_payslip.py

```diff
@@ -0,0 +1,56 @@
+from odoo import fields, models
+
+
+class HrPayslip(models.Model):
+    _inherit = "hr.payslip"
+
+    line_manually_ids = fields.One2many(
+        "hr.payslip.line.manually",
+        "res_id",
+        string="Manual Payslip Lines",
+        domain=[("model", "=", "hr.payslip")],
+        context={"default_model": "hr.payslip"},
+        copy=True,
+    )
+
+    def _compute_payslip_line(self, rule, localdict, lines_dict):
+        self.ensure_one()
+        payslip_lines = rule._compute_values(localdict)
+        if not isinstance(payslip_lines, list):
+            payslip_lines = [payslip_lines]
+        for payslip_line in payslip_lines:
+            values = payslip_line
+            key = (
```

Comment:
Are you only changing the "key" value here? If you are, why not call super() and then you can just do: 

`if values.get("_compute_payslip_line"):
`# Change the key of lines_dict